src/Functional/Connectivity/Products.php

```diff
@@ -70,20 +68,17 @@ public function getSingleProduct(string $productId): ProductDataInterface
 
     public function getProductOrderBookRaw(string $productId, int $level = self::LEVEL_ONE, bool $forceLevel3 = false): string
     {
-        $query = ['level' => 1];
-        if (2 === $level) {
-            $query['level'] = 2;
-        }
-        if (3 === $level && $forceLevel3) {
-            $query['level'] = 3;
+        if ($forceLevel3 === true) {
+            $level = 3;
+        } else {
+            $level = $level <= 3 ? $level : 1;
```

Comment:
$forceLevel3 is for double check $level = 3, cause, accaording to the cocs, repeated call to level3 can : Level 3 is only recommended for users wishing to maintain a full real-time order book using the websocket stream. Abuse of Level 3 via polling will cause your access to be limited or blocked.

Reply:
Makes sense, should rewrite it